lib/cassandro/model.rb

```diff
@@ -230,6 +230,14 @@ def self.query(where, *values)
       Cassandro.client.execute(st, *values)
     end
 
+    def self.count
```

Comment:
OK, it was my mistake. I refactored the `Cassandro::Model#count` method to accept conditional filter arguments after discussing it in person with @tarolandia.